ansible/playbook.yml

```diff
@@ -0,0 +1,52 @@
+- name: todoapp
+  hosts: franky-host
+  remote_user: ec2-user
+  vars_prompt:
+    - name: secret_token
+      private: no
+    - name: trello_token
+      private: no
+    - name: token_token
```

Comment:
Somewhat confusing variable names.
`trello_token` is used for the Trello key and `token_token` is used for the Trello token.

Note that you can clarify the meaning of the variables by setting a `prompt: ` field (alongside `name`/`private`) to give the user a message, e.g. `prompt: Enter your Trello API token`